purchases/consumer-rules.pro

```diff
@@ -1,4 +1,4 @@
--keep class com.revenuecat.** { *; }
+-keep class com.revenuecat.purchases.amazon.** {*;}
```

Comment:
Is this needed? We are already keeping them in the consumer rules of the amazon module ttps://github.com/RevenueCat/purchases-android/pull/1193/files#diff-327934b4ae63dd824b813f85b180c4689acb3a23339ecee4dbdb40d40dec32da